app/services/email_normalizer.rb

```diff
@@ -23,7 +26,7 @@ def normalized_email
   private
 
   def gmail?
-    email.domain == 'gmail.com' || google_mx_record?
+    email.domain == 'gmail.com' || google_mx_lookup && google_mx_record?
```

Comment:
let's add a predicate method reader for this so it's clearer it's a boolean? and maybe some parens also for clarity?
```suggestion
    email.domain == 'gmail.com' || (google_mx_lookup? && google_mx_record?)
```